R-package/vignettes/discoverYourData.Rmd

```diff
@@ -199,45 +192,16 @@ importance <- xgb.importance(feature_names = colnames(sparse_matrix), model = bs
 head(importance)
 ```
 
-> The column `Gain` provide the information we are looking for.
+> The column `Gain` provides the information we are looking for.
 >
 > As you can see, features are classified by `Gain`.
 
-`Gain` is the improvement in accuracy brought by a feature to the branches it is on. The idea is that before adding a new split on a feature X to the branch there was some wrongly classified elements, after adding the split on this feature, there are two new branches, and each of these branch is more accurate (one branch saying if your observation is on this branch then it should be classified as `1`, and the other branch saying the exact opposite).
+`Gain` is the improvement in accuracy brought by a feature to the branches it is on. The idea is that before adding a new split on a feature X to the branch there were some wrongly classified elements; after adding the split on this feature, there are two new branches, and each of these branches is more accurate (one branch saying if your observation is on this branch then it should be classified as `1`, and the other branch saying the exact opposite).
 
-`Cover` measures the relative quantity of observations concerned by a feature.
+`Cover` measures the relative percent of observations concerned by a feature.
 
 `Frequency` is a simpler way to measure the `Gain`. It just counts the number of times a feature is used in all generated trees. You should not use it (unless you know why you want to use it).
 
-#### Improvement in the interpretability of feature importance data.table
```

Comment:
Any chance we can revise this part instead of removing it entirely? We still have the properties representing global feature importance, including `weight`, `gain`, `cover`, `total_gain`, and `total_cover`.

Reply:
Got it, let's remove it.